doc/shells.rst

```diff
@@ -55,6 +55,13 @@ PuDB supports the following external shells.
 - `bpython <https://bpython-interpreter.org/>`_
 - `ptpython <https://github.com/jonathanslenders/ptpython>`_
 
+The `IPython` shell can also be used in a server-client fasion, which is
+enabled by selecting the shell `ipython_kernel` in the settings. When set, the
+``!`` key will start an `IPython` kernel and wait for connection from
+`qtconsole`. Like other shells, `ipython_kernel` blocks the debugger UI while
+it is active. Send "quit" from a client to exit the kernel and return to the
```

Comment:
How does one send "quit" from the Qt console?

https://qtconsole.readthedocs.io/en/stable/ does not shed any light.

Reply:
It just means typing `quit<enter>` from a connected qtconsole. It is an IPython thing https://ipython.readthedocs.io/en/stable/api/generated/IPython.core.autocall.html#IPython.core.autocall.ExitAutocall. Will make the doc on this more clear.

Reply:
Changed the wording from "Send" to "Type" to avoid possible confusion in b4d73d3